functions/Get-DbaXEventSessionTarget.ps1

```diff
@@ -0,0 +1,143 @@
+﻿function Get-DbaXEventSessionTarget {
+ <#
+	.SYNOPSIS
+	Get a list of Extended Events Session Targets
+
+	.DESCRIPTION
+	Retrieves a list of Extended Events Session Targets
+	
+	.PARAMETER SqlInstance
+	SQL Server name or SMO object representing the SQL Server to connect to. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
+
+	.PARAMETER SqlCredential
+	SqlCredential object to connect as. If not specified, current Windows login will be used.
+
+	.PARAMETER Session
+	Only return a specific session. This parameter is auto-populated.
+	
+	.PARAMETER Target
+	Only return a specific target. 
+	
+	.PARAMETER SessionObject
+	Internal pipeline parameter
+	
+	.PARAMETER Silent
+	If this switch is enabled, the internal messaging functions will be silenced.
+
+	.NOTES
+	Tags: Xevent
+	Website: https://dbatools.io
+	Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+	License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+	
+	.LINK
+	https://dbatools.io/Get-DbaXEventSessionTarget
+	
+	.EXAMPLE
+	Get-DbaXEventSessionTarget -SqlInstance ServerA\sql987 -Session system_health
+
+	Shows targets for the system_health session on ServerA\sql987
+
+	.EXAMPLE
+	Get-DbaXEventSession -SqlInstance sql2016 -Session system_health | Get-DbaXEventSessionTarget
+	
+	Returns the targets for the system_health session on sql2016
+	
+	.EXAMPLE
+	Get-DbaXEventSession -SqlInstance sql2016 -Session system_health | Get-DbaXEventSessionTarget -Target package0.event_file
+	
+	Return only the package0.event_file target for the system_health session on sql2016 
+#>
+	[CmdletBinding(DefaultParameterSetName = "Default")]
+	param (
+		[parameter(ValueFromPipeline, ParameterSetName = "instance", Mandatory)]
+		[Alias("ServerInstance", "SqlServer")]
+		[DbaInstanceParameter[]]$SqlInstance,
+		[PSCredential]$SqlCredential,
+		[string[]]$Session,
+		[string[]]$Target,
+		[parameter(ValueFromPipeline, ParameterSetName = "piped", Mandatory)]
+		[Microsoft.SqlServer.Management.XEvent.Session[]]$SessionObject,
+		[switch]$Silent
+	)
+	
+	begin {
+		if ([System.Reflection.Assembly]::LoadWithPartialName("Microsoft.SqlServer.Management.XEvent") -eq $null) {
+			Stop-Function -Message "SMO version is too old. To collect Extended Events, you must have SQL Server Management Studio 2012 or higher installed."
+		}
+		
+		function get-target ($Sessions, $server) {
```

Comment:
I know it's an internal function, but still, it should have a `[CmdletBinding()]` (and thus a param block) anyway
Also watch your casing (it's upper-case when you later call it)

Reply:
fixed